For XanMod kernel:

Revert the pre-action, after applying the realtime patch set.

diff -uar a/kernel/rcu/rcutorture.c b/kernel/rcu/rcutorture.c
--- a/kernel/rcu/rcutorture.c
+++ b/kernel/rcu/rcutorture.c
@@ -2408,7 +2408,7 @@
 		t = per_cpu(ksoftirqd, cpu);
 		WARN_ON_ONCE(!t);
 		sp.sched_priority = 2;
-		sched_setscheduler_nocheck(t, SCHED_FIFO, &sp);
+		sched_setscheduler_nocheck(t, SCHED_RR, &sp);
 #ifdef CONFIG_PREEMPT_RT
 		t = per_cpu(timersd, cpu);
 		WARN_ON_ONCE(!t);
